inventory/sample/group_vars/k8s-cluster/addons.yml

```diff
@@ -122,6 +122,8 @@ cert_manager_enabled: false
 
 # MetalLB deployment
 metallb_enabled: false
+# metallb_speaker_image_repo: "{{ docker_image_repo }}/metallb/speaker"
+# metallb_controller_image_repo: "{{ docker_image_repo }}/metallb/controller
```

Comment:
There is not any other addons which contain `docker_image_repo` under inventory/sample.
Is this necessary for metallb?

Reply:
@oomichi The reason behind making this change for metallb is for installing it on a cluster without internet access. If there is a different or preferred way to support this I can update.

Reply:
@dlouks Sorry, my previous comment was not clear.
I don't have a question about this PR itself.
I have a question about adding the above lines into inventory/sample/group_vars/k8s-cluster/addons.yml because there are not any lines which contain `docker_image_repo` under `inventory/sample/` directory.

Reply:
I see what you mean. I've removed those commented lines from the addon.yml.